cradle-core/src/main/java/com/exactpro/cradle/intervals/IntervalsWorker.java

```diff
@@ -38,18 +40,38 @@
     CompletableFuture<Boolean> storeIntervalAsync(Interval interval);
 
     /**
-     * Obtains iterable of intervals with startTime greater or equal that from and less or equal then to
-     * @param from time from which intervals are being searched
+     * Obtains iterable of intervals with startTime greater or equal that from and less or equal then to. Intervals must be within one day
+     * @param from time from which intervals are searched
+     * @param to time to which intervals are searched
+     * @param crawlerName name of Crawler
+     * @param crawlerVersion version of Crawler
+     * @param crawlerType type of Crawler
      * @return iterable of intervals
      */
-    Iterable<Interval> getIntervals(Instant from, Instant to, String crawlerName, String crawlerVersion, String crawlerType) throws IOException;
+    Iterable<Interval> getIntervalsPerDay(Instant from, Instant to, String crawlerName, String crawlerVersion, String crawlerType) throws IOException;
 
     /**
-     * Asynchronously obtains iterable of intervals with startTime greater or equal that from and less or equal then to
-     * @param from time from which intervals are being searched
+     * Asynchronously obtains iterable of intervals with startTime greater or equal that from and less or equal then to. Intervals must be within one day
```

Comment:
Replace this line this the following:
`* Asynchronously obtains iterable of intervals with startTime greater than or equal to "from" and less than or equal to "to". Intervals must be within one day`